### PR TITLE
Add links to CCG from S2S guides

### PR DESCRIPTION
**Pull request by joelhans**
Following up on #1008 by adding a few cross-links from existing guides.

Very much open to feedback on the copy itself or its positioning within the S2S guides.

### CODE REVIEW

File: docs/faq/faq.md [hunk 1]
```diff
@@ -155,3 +155,12 @@ IP and port to indicate which customer to route traffic to.
 As with all publicly accessible IPs and ports on the internet anyone can send
 traffic to those endpoints, but ngrok provides a [number of mechanisms](https://ngrok.com/docs/guides/security-dev-productivity/) to control
```
Comment (DevMandy) [inline docs/faq/faq.md:156]: traffic to those endpoints, but ngrok provides a [number of mechanisms](/docs/other-guides/security-dev-productivity/) to control

Comment (DevMandy) [reply docs/faq/faq.md:156]: Update the link

File: docs/faq/faq.md [hunk 2]
```diff
@@ -155,3 +155,12 @@ IP and port to indicate which customer to route traffic to.
...
 what traffic actually gets routed to you ngrok agent or sdk.
+
+## Why is my vendor asking me to install ngrok for 'site-to-site connectivity'? {#site-to-site-end-customer}
+
+Your vendor wants to create a secure persistent connection between your network
+and theirs, which allows them to access and take action on your services and
+data.
+
+For details about what ngrok is, how to install the ngrok agent, and how to work
+with your vendor to implement adequate security practices, see our [end customer installation guide](end-customers.mdx).
```
Comment (DevMandy) [inline docs/faq/faq.md:166]: with your vendor to implement adequate security practices, see our [end customer installation guide](/docs/guides/site-to-site-connectivity/end-customers/).

**Review by DevMandy — CHANGES_REQUESTED** [review]

**Comment by joelhans** [comment]
@DevMandy Fixed both of those. TYVM!

**Review by DevMandy — APPROVED** [review]

**Pull request merged**